File: ReportingApplication/ReportingApplication/Models/Attachment.cs
namespace ReportingApplication.Models
{
    public class Attachment(string fileName, string filePath, Guid ticketId)
    {
        public int Id { get; set; }
        public string FileName { get; set; } = fileName;
        public string FilePath { get; set; } = filePath;
        public Guid TicketId { get; set; } = ticketId;
        public Ticket ticket { get; set; } = null!;
    }
}

File: ReportingApplicationFrontend/src/app/models/attachment.ts
export class Attachment{
    id: number;
    fileName: string;
    filePath: string;
    ticketId: string;

    constructor(_id: number, _fileName: string, _filePath: string, _ticketId: string){
        this.id = _id;
        this.fileName = _fileName;
        this.filePath = _filePath;
        this.ticketId = _ticketId;
    }
}